examples/platform/esp32/common/Esp32AppServer.cpp

```diff
@@ -111,47 +111,12 @@ static size_t hex_string_to_binary(const char * hex_string, uint8_t * buf, size_
 
 void Esp32AppServer::DeInitBLEIfCommissioned(void)
 {
-#if CONFIG_BT_ENABLED && CONFIG_USE_BLE_ONLY_FOR_COMMISSIONING
+#if CONFIG_USE_BLE_ONLY_FOR_COMMISSIONING
```

Comment:
Why is this removed? As `CONFIG_USE_BLE_ONLY_FOR_COMMISSIONING` depends on `CONFIG_BT_ENABLED` ?

Reply:
Yes.